### DISCUSSION TITLE
Cost of hosting your own battle

### DISCUSSION THREAD

**Discussion by SamBenson** [cssbattle/forum#155] [General]
Why is there such a high price on hosting your own battle? I can't help but feel that if there was a much more accessible entry to hosting then more development teams (like my own) would be interested in creating battles internally and using them more regularly. 

The reason I ask is that I was about to set up a battle for my team but the thought of paying $199 for something that is effectively just a bit of fun just wasn't going to happen. 

Even if you aren't able to rethink the price, It would be great if, as a pro member, you were able to create private battles that you can share with friends.

**Comment by Beowolve** [comment]
I fully agree with that, the price is way too high and I bet there have been very few custom battles so far if any.
There could have been a battle between friends for me if the price was reasonable. If I could host my own private battles as a pro member I would definitely subscribe just for that.

**Reply by chinchang** [reply]
private battles in PRO membership would mean someone can buy a PRO membership for 5$/month and host a custom battle which is worth $199 as of now 😅

**Comment by chinchang** [comment]
Hey @SamBenson & @Beowolve 👋🏼 
Thanks for putting in your thoughts here! 
Pricing is always a tricky thing and an iterative process to land upon the most optimum one.
If we were to ask you, whats that price which you would happily pay for a custom battle, what would it be?

**Reply by bgalek** [reply]
Hi, just discovered cssbattle.dev, It's super awesome! Good job!

We're having an onsite team meeting during xmas season and I wanted to host a custom battle for ~10 people for fun.
$199 is a pretty big investment for first-timers (I won't spend that amount of money not knowing if people are going to enjoy it as much as I do). This discussion is about pricing - in my case I could spend something like $10-$15 for one-time event.

I know pricing is hard, but maybe you guys should think about tiers? Like: custom battle 2-3 participants - $15, 3-10 participants $25$ and so on. The same goes for time limit - in my case 1 day is enought. If somebody wants to make it 30 days long - price could be adjusted. Or maybe one free custom battle/month - limited to i.e. 10 minutes, only time challenge, only predefined shapes, max 2 people.

Don't know if it's helpful, just sharing my point of view ;)
